### PR TITLE
feat: :sparkles: Add session storage, add updateFlightInfo and addFlightInfo functions, add history button

### PR DESCRIPTION
**Pull request by sechaba-mache**

### CODE REVIEW

File: dom-manip.js [hunk 1]
```diff
@@ -28,10 +37,12 @@ const flightCategories = [
 	"Line Obstacle",
 ];
 
+localStorage.setItem("flights", JSON.stringify([]));
```
Comment (kphutiyagae) [inline dom-manip.js]: Separation of concerns. Please separate.

**Review by kphutiyagae — APPROVED** [review]
Please correct before next PR.

File: dom-manip.js (hunk 1 above)
Comment (sechaba-mache) [reply dom-manip.js]: Thank you. Issue has been resolved. Please confirm

**Review by kphutiyagae — APPROVED** [review]
Please reconsider naming conventions.

**Comment by sechaba-mache** [comment]
> Please reconsider naming conventions.

Will be done with addition of observables

**Pull request merged**